package.json

```diff
@@ -3,6 +3,7 @@
     "type": "git",
     "url": "http://github.com/cfpb/cfgov-refresh.git"
   },
+  "license": "CC0-1.0",
```

Comment:
Yup! Fixed in https://github.com/cfpb/cfgov-refresh/commit/7f94cd21873cb4f07ee3a1c09fde14cd0056f0f6